### PR TITLE
Fix deluge configuration

### PR DESCRIPTION
**Pull request by cgtobi**
**Description:**
Fix deluge configuration.

Related to #6385.

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
@cgtobi The build fails, could you take a look? 👍

**Comment by cgtobi** [comment]
Looks like it was the parentheses which needed double quotes.

File: source/_components/sensor.deluge.markdown [hunk 1]
```diff
@@ -31,14 +31,37 @@ sensor:
...
+  required: false
+  type: integer
+  description: The port your Deluge daemon uses. Defaults to 58846. Warning, this is not the port of the WebUI.
+name:
+  required: false
+  type: string
+  default: Deluge
+  description: The name to use when displaying this Deluge instance.
+username:
+  required: false
```
Comment (WofWca) [inline source/_components/sensor.deluge.markdown]: Isn't it required?

**Review by WofWca — CHANGES_REQUESTED** [review]

File: source/_components/sensor.deluge.markdown (hunk 1 above)
Comment (cgtobi) [reply source/_components/sensor.deluge.markdown]: Oops. Thank you.

**Review by fabaff — APPROVED** [review]
Thanks :bird:

**Pull request merged**